### PR TITLE
DLS-8640 Delete Admin Routes

### PR DESCRIPTION
**Pull request by LAJW**

### CODE REVIEW

File: app/uk/gov/hmrc/cbcr/services/AdminService.scala [hunk 1]
```diff
@@ -65,80 +65,6 @@ class AdminService @Inject()(
   def displayAllDocRefId(docs: List[DocRefIdRecord]): ListDocRefIdRecord =
     ListDocRefIdRecord(docs)
```
Comment (johno1985) [inline app/uk/gov/hmrc/cbcr/services/AdminService.scala:67]: do we not want to delete the entire service?

**Review by johno1985 — COMMENTED** [review]
minor question

File: app/uk/gov/hmrc/cbcr/services/AdminService.scala (hunk 1 above)
Comment (LAJW) [reply app/uk/gov/hmrc/cbcr/services/AdminService.scala:67]: This bit of code is used in another controller for legitimate reasons. Do you think we should perhaps inline it?

Comment (johno1985) [reply app/uk/gov/hmrc/cbcr/services/AdminService.scala:67]: Ah OK, perhaps, no strong opinion either way

Comment (LAJW) [reply app/uk/gov/hmrc/cbcr/services/AdminService.scala]: Turned out this is an unused admin route and doesn't have `/admin/` in the URL. Deleted.

**Review by johno1985 — APPROVED** [review]

**Pull request merged**